### PR TITLE
POM and code cleanup

### PR DESCRIPTION
**Pull request by res0nance**
- Fixup pom with https links
- Some minor code cleanups included

### CODE REVIEW

File: src/main/java/com/sonyericsson/rebuild/Rebuilder.java [hunk 1]
```diff
@@ -48,15 +48,15 @@ public Rebuilder() {
 
     @Override
     public void onCompleted(Run build, TaskListener listener) {
-            for (RebuildValidator rebuildValidator : Hudson.getInstance().
-                    getExtensionList(RebuildValidator.class)) {
-                if (rebuildValidator.isApplicable(build)) {
-                    return;
-                }
+        for (RebuildValidator rebuildValidator : Jenkins.getInstance().
+                getExtensionList(RebuildValidator.class)) {
```
Comment (oleg-nenashev) [inline src/main/java/com/sonyericsson/rebuild/Rebuilder.java]: no need in line break

File: src/main/java/com/sonyericsson/rebuild/RebuildAction.java [hunk 2]
```diff
@@ -296,20 +296,13 @@ public void doConfigSubmit(StaplerRequest req, StaplerResponse rsp) throws Servl
...
             for (ParameterValue source : paramAction.getParameters()) {
-                boolean alreadyAdded = false;
-                for (ParameterValue dest : values) {
-                    if (source.getName().equals(dest.getName())) {
-                        alreadyAdded = true;
-                        break;
-                    }
-                }
-                if (!alreadyAdded) {
+                if (!values.contains(source)) {
```
Comment (oleg-nenashev) [inline src/main/java/com/sonyericsson/rebuild/RebuildAction.java]: Not sure it will work properly for all `ParameterValue ` implementations.

Comment (res0nance) [reply src/main/java/com/sonyericsson/rebuild/RebuildAction.java]: I was unsure why the previous code was so roundabout but that makes sense why it is. Reverted this bit.

File: src/main/java/com/sonyericsson/rebuild/Rebuilder.java (hunk 1 above)
Comment (res0nance) [reply src/main/java/com/sonyericsson/rebuild/Rebuilder.java]: The original was in 2 lines but I merged it to 1

**Comment by res0nance** [comment]
Retrigger CI

**Comment by oleg-nenashev** [comment]
hm? Did you just drop it @res0nance ?

**Comment by res0nance** [comment]
@oleg-nenashev  I messed up the history pretty bad and #59 already got the POM bits so I'll reopen this with just the code bits instead

**Comment by oleg-nenashev** [comment]
ack, thanks!

**Pull request closed without merging**